extensions/amp-mustache/amp-mustache.md

```diff
@@ -16,7 +16,50 @@ limitations under the License.
 
 ### <a name="amp-mustache"></a> `amp-mustache`
 
-The `amp-mustache` allows rendering of [Mustache.js](https://github.com/janl/mustache.js/) templates.
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Allows rendering of <a href="https://github.com/janl/mustache.js/">Mustache.js</a>.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Stable</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td>None</td>
+  </tr>
+</table>
+
+The following lists validation errors specific to the `amp-mustache` tag
+(see also `amp-mustache` in the [AMP validator specification](https://github.com/ampproject/amphtml/blob/master/validator/validator.protoascii):
+
+<table>
+  <tr>
+    <th width="40%"><strong>Validation Error</strong></th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td width="40%"><a href="/docs/reference/validation_errors.html#tag-required-by-another-tag-is-missing">TAG_REQUIRED_BY_MISSING</a></td>
+    <td>Error thrown when required <code>amp-mustache</code> extension <code>.js</code> script tag is missing or incorrect.</td>
+  </tr>
+  <tr>
+    <td width="40%"><a href="/docs/reference/validation_errors.html#attribute-contains-template-syntax">TEMPLATE_IN_ATTR_NAME</a></td>
+    <td>Mustache templates must be within the <code>amp-mustache</code> tag. Error thrown when templating syntax found in an attribute in another tag.</td>
+  </tr>
+  <tr>
+    <td width="40%"><a href="/docs/reference/validation_errors.html#attribute-contains-unescaped-template-syntax">UNESCAPED_TEMPLATE_IN_ATTR_VALUE</a></td>
```

Comment:
A useful reference link for "unescaped templating syntax" might be https://github.com/janl/mustache.js/#variables